.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: julia
 julia:
-  - 0.6
```

Comment:
The package still supports 0.6

Reply:
not anymore, I used 1.0-only features :P

but seriously, do future versions need to still support 0.6?

Reply:
No, but the REQUIRE file should be updated to reflect that

Reply:
Ah, ok, I forgot about that one. I think I got a few more references too.